README.rst

```diff
@@ -23,41 +23,252 @@ Log collector service
             encoder: es_json
             message_matcher: TRUE
 
+
 Metric collector service
+------------------------
+
+Local alarm definition for nova compute role
 
 .. code-block:: yaml
 
     heka:
       metric_collector:
-        enabled: true
-        output:
-          elasticsearch01:
-            engine: elasticsearch
-            host: localhost
-            port: 9200
-            encoder: es_json
-            message_matcher: TRUE
-          dashboard01:
-            engine: dashboard
-            ticker_interval: 30
+        filter:
+          nova_compute_filesystem_warning:
+            engine: afd_trigger
+            enabled: True  # implicit
+            description: "The nova instance filesystem's root free space is low."
+            severity: warning
+            logical_operator: or # implicit
+            rules:
+            - metric: fs_space_percent_free
+              relational_operator: '<'
+              threshold: 10
+              window: 60
+              periods: 0
+              function: min
+              fs: '/var/lib/nova'
+          nova_compute_filesystem_critical:
+            engine: afd_trigger
+            enabled: True  # implicit
+            description: "The nova instance filesystem's root free space is low."
+            severity: warning
+            logical_operator: or # implicit
+            rules:
+            - metric: fs_space_percent_free
+              relational_operator: '<'
+              threshold: 5
+              window: 60
+              periods: 0
+              function: min
+              fs: '/var/lib/nova'
+          nova_compute_service:
+            engine: afd_alarm
+            notifications: False
```

Comment:
notifications and alerting properties are currently expressed by only one property:

alerting: disabled|enabled|enabled_with_notification